From 2d41dab2f23efdc5cace917ffdcd60cb7ca77525 Mon Sep 17 00:00:00 2001
From: irqchip-bot for Yanteng Si <tip-bot2@linutronix.de>
Date: Thu, 28 Oct 2021 13:23:24 +0000
Subject: [PATCH 15/22] MIPS: irq: Avoid an unused-variable error

The following commit has been merged into the irq/irqchip-next branch of irqchip:

Commit-ID:     34fca8947b2743e6a3a9a8a3a44962e625993533
Gitweb:        https://git.kernel.org/pub/scm/linux/kernel/git/maz/arm-platforms/34fca8947b2743e6a3a9a8a3a44962e625993533
Author:        Yanteng Si <siyanteng01@gmail.com>
AuthorDate:    Thu, 28 Oct 2021 17:56:52 +08:00
Committer:     Marc Zyngier <maz@kernel.org>
CommitterDate: Thu, 28 Oct 2021 13:28:52 +01:00

MIPS: irq: Avoid an unused-variable error

When CONFIG_IRQ_DOMAIN is set, there is a warning:
arch/mips/kernel/irq.c:114:19: error: unused variable 'desc' [-Werror=unused-variable]
  114 |  struct irq_desc *desc;
      |                   ^~~~

This variable is unused, let's remove it.

Signed-off-by: Yanteng Si <siyanteng@loongson.cn>
Signed-off-by: Marc Zyngier <maz@kernel.org>
Link: https://lore.kernel.org/r/20211028095652.3503790-1-siyanteng@loongson.cn
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
Link: https://lore.kernel.org/r/163542740410.626.5186631730404597986.tip-bot2@tip-bot2
---
 arch/mips/kernel/irq.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/mips/kernel/irq.c b/arch/mips/kernel/irq.c
index 1fee96ef8059..5e11582fe308 100644
--- a/arch/mips/kernel/irq.c
+++ b/arch/mips/kernel/irq.c
@@ -111,8 +111,6 @@ void __irq_entry do_IRQ(unsigned int irq)
 #ifdef CONFIG_IRQ_DOMAIN
 void __irq_entry do_domain_IRQ(struct irq_domain *domain, unsigned int hwirq)
 {
-	struct irq_desc *desc;
-
 	irq_enter();
 	check_stack_overflow();
 	generic_handle_domain_irq(domain, hwirq);
-- 
2.33.1.637.gf443b226ca

